examples/serialization.py

```diff
@@ -0,0 +1,122 @@
+# Copyright © 2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+"""
+Example of serializing and de-serializing a surrogate model using JSON and pickling methods
```

Comment:
Question: Does this being a surrogate model add anything to the example. I wondering if we should just do the ElectroChem instead. What are your thoughts

Reply:
I agree, much simpler with a standard battery model. Made this change.